tools/cdep/app/update.go

```diff
@@ -167,15 +168,22 @@ func (a App) Update(ctx context.Context, req *parsers.Params, overruleChecks []s
 	commitMessage := fmt.Sprintf("cdep: %s", req.String("update"))
 
 	if !a.DryRun && req.Environment == "prod" {
-		textTemplate := ":bell: A prod deployment is happening :bell:\n\n:wrench: *command*: `%s`\n:technologist: *user*: `%s`"
+		textTemplate := ":wrench: *command*: `%s`\n:technologist: *user*: `%s`"
 		text := fmt.Sprintf(textTemplate, req.String("update"), strings.Split(string(user), "\n")[0])
 		if req.Message != "" {
 			text = text + fmt.Sprintf("\n\n:email: *message*: `%s`", req.Message)
 		}
 
-		slackMessage := &slack.WebhookMessage{Text: text}
-
-		slack.PostWebhookContext(ctx, "https://hooks.slack.com/services/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", slackMessage)
+		arn := "arn:aws:sns:eu-west-1:005717268539:cuvva-deployments-prod"
+		subject := "A prod deployment is happening"
+		_, err := a.SNS.PublishWithContext(ctx, &sns.PublishInput{
+			TopicArn: &arn,
+			Subject:  &subject,
+			Message:  &text,
+		})
+		if err, ok := err.(awserr.Error); !ok || err.Code() != "EndpointDisabled" {
+			fmt.Println(err)
```

Comment:
Could be worth wrapping this error in some way? As the direct AWS one may be a little confusing

Reply:
It's not that confusing. I had it printed a few times. `AuthorizationError: User: arn:aws:sts::814389279259:assumed-role/cuvva-admin-role/1611142113196353000 is not authorized to perform: SNS:Publish on resource: arn:aws:sns:eu-west-1:005717268539:cuvva-deployments-prod` this is one example of how they look.